build/ferretdb/evaluation/supervisord.conf

```diff
@@ -0,0 +1,20 @@
+[supervisord]
```

Comment:
> CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message. 

So why don't we add `user=root`?

Reply:
added